lib.php

```diff
@@ -15,71 +15,99 @@
 // along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
 
 /**
- * Tiny C4L library functions.
+ * Tiny Elements library functions.
  *
- * @package   tiny_c4l
+ * @package   tiny_elements
  * @copyright 2023 Marc Català <reskit@gmail.com>
  * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
  */
 
-defined('MOODLE_INTERNAL') || die();
+use tiny_elements\local\utils;
 
 /**
- * Return a list of all the user preferences used by tiny_c4l.
+ * Return a list of all the user preferences used by tiny_elements.
  *
  * @return array
  */
-function tiny_c4l_user_preferences() {
+function tiny_elements_user_preferences() {
```

Comment:
hasn't user_preferences already migrated to hooks? If yes, please migrate, if no, it's fine of course :)